jenkins/spark-premerge-build.sh

```diff
@@ -109,6 +109,10 @@ mvn_verify() {
     do
         TZ=$tz ./integration_tests/run_pyspark_from_build.sh -m tz_sensitive_test
     done
+
+    # test Hybrid feature
+    source "${WORKSPACE}/jenkins/hybrid_execution.sh"
```

Comment:
how long will this test take? we may need some duration to determine if we should put it into in ci1 or ci2 stage for balancing the workloads